### PR TITLE
Add backend example of using client session 

### PR DESCRIPTION
**Pull request by razor-x**
This reverts commit dfc7cfa5dbf28f3d36bad092616d2d8d723425b9.

### CODE REVIEW

File: docs/api-clients/client-sessions/README.md [hunk 1]
```diff
@@ -42,3 +42,75 @@ You can perform the following actions on `client_session` objects:
...
+  const connect_webview = await seam.connectWebviews.create()
+
+  // Link the Connect Webview to the client session.
+  await seam.clientSessions.grant_access({
+    user_identifier_key,
+    connect_webview_ids: [connect_webview.connect_webview_id],
+  })
+
+  // Redirect the user to the Connect Webview.
+  res.redirect(connect_webview.url)
```
Comment (dawnho) [inline docs/api-clients/client-sessions/README.md:81]: I don't know if we need to show the redirect here. We can just say to use the url to let the end user connect their account

File: docs/api-clients/client-sessions/README.md [hunk 2]
```diff
@@ -42,3 +42,75 @@ You can perform the following actions on `client_session` objects:
...
+}
+
+
+const handleUserDeviceList = async (req, res) => {
+  const user_identifier_key = req.auth.userId
+
+  const devices = await seam.devices.list({
+    user_identifier_key,
```
Comment (dawnho) [inline docs/api-clients/client-sessions/README.md:89]: what is `user_identifier_key`? Is this from the client session?

Comment (razor-x) [reply docs/api-clients/client-sessions/README.md:89]: It's the customer application's user id, in the example it's read from the auth context

**Comment by razor-x** [comment]
Closing in favor of new CR in gitbook

**Pull request closed without merging**